### ISSUE TITLE
Moving statistics to separate Handler

### ISSUE THREAD

**Issue by Shin-nn** [mattgodbolt/seasocks#8]
Wouldn't be better and cleaner solution to move statistis to separate Handler? Than it would be more secure to use server without possibility of leaking info out.

**Comment by mattgodbolt** [comment]
Agreed, though please do note that security and information leakage avoidance are not really Seasocks's goals.

**Comment by Shin-nn** [comment]
Yeah I know, but i wen through few implementations of WebServer with WebSockets and this is the first, that is possible to use.

**Comment by mattgodbolt** [comment]
Thanks! Glad it works for you; we use Seasocks in a semi-secure context but I'd hate to think anyone thought it had been designed for security :)
